src/lib/components/PathField.svelte

```diff
@@ -9,6 +9,7 @@
 	export let icon: string = 'mdi:folder';
 
 	$: hasValue = value && value.length > 0;
+	$: valueExcessive = value && value.length > 24;
```

Comment:
That still doesn't get rid of the problem. RTL rendering affects "Other Neutral" Unicode characters.
It's best to keep `direction: rtl` and use the Right-to-Left Mark ([U+200E](https://graphemica.com/200E)) to suppress the bidi rendering.

Not sure why you opened another PR. Any changes that need to be done can be made in #100.

Reply:
> That still doesn't get rid of the problem. RTL rendering affects "Other Neutral" Unicode characters. It's best to keep `direction: rtl` and use the Right-to-Left Mark ([U+200E](https://graphemica.com/200E)) to suppress the bidi rendering.
> 
> Not sure why you opened another PR. Any changes that need to be done can be made in #113.

Sorry, I was in a hurry.